tests/MulticastUDPClient.py

```
@@ -111,68 +111,19 @@
-		import random
-	else:  # pragma: no branch
-		random = sys.modules["""random"""]
-except Exception as badErr:  # pragma: no branch
-	baton = ModuleNotFoundError(badErr, str("[CWE-758] Test module failed completely."))
-	baton.module = __module__
-	baton.path = __file__
-	baton.__cause__ = badErr
-	raise baton from badErr
+	import random
+except ImportError as baton:  # pragma: no branch
+	raise ModuleNotFoundError(baton, str("[CWE-758] Test module failed to randomize.")) from baton
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)